### PR TITLE
update:ci:deprecate setup_wince.sh script

### PR DESCRIPTION
**Pull request by aerostitch**
Follow-up of https://github.com/navit-gps/Dockerfiles/pull/3

### CODE REVIEW

**Comment by aerostitch** [comment]
Looks like installing `git-core` brings a bad version of git that doesn't properly support to git clone in `.`.
The upgrade of the image starts to be a blocker for me. I guess I'll have to make it a priority :disappointed:

File: scripts/setup_wince.sh [hunk 1]
```diff
@@ -1,4 +1,10 @@
 #!/bin/bash
+
+# #############################################################
```
Comment (jkoan) [inline scripts/setup_wince.sh]: please put this in echo so it also print out when this script runs

Comment (aerostitch) [reply scripts/setup_wince.sh]: Done.

**Comment by aerostitch** [comment]
Scanning dependencies of target git_version
-- git not found, cannot record git commit
-- Git commit: xdevxgitxnotxfound
-- Git   date: 7001010000
[  0%] Built target git_version

**Comment by jkoan** [comment]
@pgrandin yes. But I head the idea to add a fake gut program in the path witch handles the "." directory by checking out to a temp directory and then move everything once finished

**Comment by aerostitch** [comment]
I'll close this one until I get more time to work on the image upgrade for wince as it's quite dependent on it to work properly.

**Pull request closed without merging**